src/pages/trade/model/useSummary.ts

```diff
@@ -24,3 +24,24 @@ export const useSummary = (window: DurationWindow) => {
 
   return query;
 };
+
+// Fetch USDC-denominated prices for each quote asset to enable cross-pair comparisons.
+// Uses 'useQueries' hook for handling multiple dynamic numbers of queries
+export const useMultipleSummaries = (quoteAssets: string[]) => {
+  const queries = useQueries<Array<{ data: SummaryData | NoSummaryData | undefined }>>({
+    queries: quoteAssets.map(asset => ({
+      queryKey: ['multipleSummaries', asset, 'USDC'] as const,
+      queryFn: async () => {
+        return apiFetch<SummaryData | NoSummaryData>('/api/summary', {
```

Comment:
It seems that the` /api/stats` and `/api/summaries` endpoints from pindexer may be returning incorrect information, impacting metrics like "total trading volume (24h)" and the largest "trading pair (24h volume)" for the USDC/USDY pair. 

previous [discussions](https://github.com/penumbra-zone/dex-explorer/pull/133#discussion_r1856531144) suggest that this may not simply be related to the denom exponents not being handled.